Helper/Data.php

```diff
@@ -328,14 +380,30 @@ public function optInSubscriber($email, $data = [])
      */
     public function cronSubscribeAll($list)
     {
+        $subscribers = [];
         foreach ($list as $batch) {
-            $response = $this->callApi('contact', $batch, 'POST');
-            if (!array_key_exists('message', $response) ||
-                array_key_exists('message', $response) && $response['message'] !== 'OK') {
+            // Subscribe customers to each website separately.
+            foreach ($this->getWebsiteIds() as $websiteId) {
+                if (! $this->isCronEnabledForWebsite($websiteId)) {
```

Comment:
Shouldn't it also check if extension is enabled?

Reply:
Yes, forgot enable checks. Added